README.md

```diff
@@ -46,7 +46,7 @@ In order to get this, we make 4 RxJS Subjects:
 * Shell
 * STDIN
 * Control
-* IOPub (ok, this should actually just be an `Observable`)
+* IOPub (Which can alternatively be an `Observable`)
```

Comment:
Admittedly, this was me putting an informal stance on this. We should cut the parenthetical comment out given my thoughts in https://github.com/nteract/enchannel-zmq-backend/issues/1